scipy/signal/filter_design.py

```diff
@@ -3653,6 +3653,232 @@ def besselap(N, norm='phase'):
     return asarray([]), asarray(p, dtype=complex), float(k)
 
 
+def iirnotch(w0, bw, rp=3):
+    """
+    Design second-order IIR notch digital filter.
+
+    A notch filter is a band-stop filter with a narrow bandwidth
+    (high quality factor). It rejects a narrow frequency band and
+    leaves the rest of the spectrum little changed.
+
+    Parameters
+    ----------
+    w0 : float
+        Normalized frequency to remove from a signal. It is a
+        scalar that must satisfy  0 < w0 < 1, with w0 = 1 corresponding
+        to half of the sampling frequency.
+    bw : float
+        Normalized filter bandwidth (the diference between up and lower
+        passband edges).
+    rp : float, optional
```

Comment:
From MATLAB's `ellip` documentation: "... and Rs decibels of stopband attenuation down from the peak passband value." So I think `rs` is indeed a good and consistent name here.